types/ltx/lib/Element.d.ts

```diff
@@ -1,21 +1,26 @@
+import { clone } from './clone';
+
+export type Node = Element | TextNode;
+export type TextNode = string | number;
+
 /**
  * Element
  *
  * Attributes are in the element.attrs object. Children is a list of
  * either other Elements or Strings for text content.
- **/
-export declare class Element {
+ */
+export class Element {
     name: string;
-    parent: Element;
+    parent: Element | null;
```

Comment:
Intentionally `null` here but `undefined` below?

Reply:
My experience has been that a lot of code bases standardize on either `null` or `undefined` and don't mix them.  I just wanted to confirm that using both was intentional.  It sounds like it is.